torch/testing/_internal/common_methods_invocations.py

```diff
@@ -3090,6 +3117,24 @@ def gradcheck_wrapper_triangular_input(op, input, *args, upper=False, **kwargs):
                SkipInfo('TestOpInfo', 'test_supported_backward', dtypes=(torch.bfloat16, ),
                         device_type='cuda', active_if=not SM53OrLater)),
            sample_inputs_func=sample_inputs_addbmm),
+    OpInfo('baddbmm',
+           dtypes=floating_types_and(torch.half),
+           dtypesIfCPU=all_types_and_complex_and(torch.float16, torch.bfloat16),
+           dtypesIfCUDA=floating_types_and(torch.float16, torch.complex64, torch.complex128,
+                                           *[torch.bfloat16] if CUDA11OrLater else []),
+           skips=(
+               # baddbmm does not correctly warn when resizing out= inputs
+               SkipInfo('TestCommon', 'test_out'),
+               # cuda gradchecks are slow
+               # see discussion https://github.com/pytorch/pytorch/pull/47761#issuecomment-747316775
+               SkipInfo('TestGradients', 'test_fn_gradgrad', device_type='cuda'),
+               # Reference for the three skips below: https://github.com/pytorch/pytorch/issues/56508
```

Comment:
Does labeling the sample inputs with `broadcasts_input` when there is lhs broadcasting address these issues?

See https://github.com/pytorch/pytorch/blob/b66a1e00a6432c6383e5b1f3f4a61f1ec65a44a4/torch/testing/_internal/common_methods_invocations.py#L100

Reply:
yeah that works thanks!